src/lib.rs

```diff
@@ -207,18 +209,32 @@ impl Client {
 /// Controls of how to execute a task
 #[derive(Clone, PartialEq, Eq, Hash, Default)]
```

Comment:
I think we probably want to implement our own default? So that timeout doesn't default to 0 (or whatever Duration's default is)
So maybe set it to the same 305s that is in the python client?